lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -1473,8 +1490,46 @@ sub format_id {
     return $id;
 }
 
+# Return the properly formatted JSON value
+# for the maintainer column
+sub format_maintainer {
+    my ( $c, $value, $ia, $commit ) = @_;
+    
+    my $user = $c->d->rs('User');
+    my $complat_user = $user->find({username => $value});
+    ($complat_user = $user->find_by_github_login($value)) unless $complat_user;
+    my $complat_user_admin = $complat_user? $complat_user->admin : '';
+
+    
+    my %maintainer;
+    if ($complat_user) {
+        %maintainer = ( duckco => $complat_user->username );
+        my $gh_id = $complat_user->github_id;
+
+        if ($commit) {
+            # give edit permissions
+            $ia->add_to_users($complat_user) unless ($complat_user_admin || $ia->users->find($complat_user->id));
+        }
+
+        my $gh_user = $gh_id? $c->d->rs('GitHub::User')->find({github_id => $gh_id})->login : undef;
+        if ($gh_user) {
+            $maintainer{github} = $gh_user;
+        }
+    } elsif (check_github($value)) {
+        # this github account isn't tied to any duck.co account
+        # we still allow it to be listed as maintainer
+        # but can't give edit permissions
+
+        %maintainer = ( github => $value );
+    }
+
+    $value = (!keys %maintainer)? undef : to_json \%maintainer;
```

Comment:
```Perl
return %maintainers ? to_json \%maintainer : undef;
```
Probably better to return 0 rather than undef.  Then you don't need `defined $x` everywhere to avoid warnings but can just do `return unless $value = format_maintainer()`.